python/paddle/fluid/layers/control_flow.py

```diff
@@ -912,6 +913,71 @@ def _complete(self):
                    "is_test": self.is_test})
 
 
+def while_loop(cond, body, loop_vars, name=None):
+    """
+    This while_loop operator control. Repeat while_loop body until cond return False.
+
+    Args:
+        cond(Callable): A callable returning a boolean tensor controlling whether to continue looping.
+	body(Callable): A callable returning a tuple, namedtuple or list of tensors of the same arity(length and
+            structure) and types as `loops_vars`.
+	loop_vars(list|tuple): A list ,namedtuple or tuple of tensors that is passed to both `cond` and `body`.
+	is_test(bool, optional): A flag indicating whether execution is in test phase. Default is None.
+	name(str, optional): Normally there is no need for users to set this property. For more information, please
+            refer to :ref:`api_guide_Name`. Default is None.
+    Examples:
+        .. code-block:: python
+
+	import paddle.fluid as fluid
+        import paddle.fluid.layers as layers
+
+	i = layers.fill_constant(shape=[1], dtype='int64', value=0)
+	ten = layers.fill_constant(shape=[1], dtype="int64", value=10)
+
+	def cond(i):
+	    return layers.less_than(i, ten)
+
+	def body(i):
+	    return layers.increment(x=i, value=1, in_place=True)
```

Comment:
Can we do it without increment inside body now? For example, can we make the body

return i + 1

Reply:
Has been modified into "return layers.elementwise_add(x=i, y=one)"